doc/Home.md

```diff
@@ -64,10 +64,6 @@ description.
 configuration. With this overview you may find unused keys, which can be used
 for non-bug.n hotkeys or to reassign bug.n functions.
 
-##### [Xmonad hotkeys](./Xmonad-hotkeys.md)
-> A list of hotkeys, which are similar to those used in
-[xmonad](http://xmonad.org/), a dynamically tiling X11 window manager on *n?x.
-
```

Comment:
It seems that this file no longer exists, and is contained in the following file

Reply:
agree.. I noticed this too. The xmonad config file is not in here.. but there are config files in the user suggested hotkeys section..